app/src/main/java/com/concordium/wallet/ui/account/common/NewAccountViewModel.kt

```diff
@@ -403,6 +408,7 @@ open class NewAccountViewModel(application: Application) :
         account.id = accountId.toInt()
         // Also save a recipient representing this account
         recipientRepository.insert(Recipient(account))
+        updateNotificationsSubscriptionUseCase()
```

Comment:
This must not be done here. At this point, the account is only submitted and is not yet active. Its finalization may fail. Processing of successfully created (finalized) accounts is done in `AccountUpdater`. Check out where `DefaultFungibleTokensManager.addForAccount` is called